### ISSUE TITLE
Volume slider is only visible by hovering over the loudspeaker icon 

### ISSUE THREAD

**Issue by cgruber-uos** [polimediaupv/paella-ethz#6]
To change the playback volume of the video it is currently a bit complicated: Default the volume slider is not visible. 

![2022-12-12 13_16_48-Paella Player 7 – Mozilla Firefox](https://user-images.githubusercontent.com/101632236/207043618-fda72df5-c002-4402-922c-f795e623e835.png)

You have to move your mouse pointer over the loudspeaker icon, then right side appears the volume slider and you can adjust the volume: 

![2022-12-12 13_17_11-Paella Player 7 – Mozilla Firefox](https://user-images.githubusercontent.com/101632236/207043630-2312b7e9-e1d9-4f5f-83e6-6e7821f5c191.png)

Also in case you resize the bowser, the space for the volume slider is not used for any other purpose but still kept free: 

![2022-12-12 13_25_33-New Issue · polimediaupv_paella-ethz – Mozilla Firefox](https://user-images.githubusercontent.com/101632236/207044659-354a3347-7945-453a-9109-eac4a9f2842d.png)

**So I see no need to fading the silder out in case it is not in "hovered" by the mouse. We should show the volume slider all the time, not only with mouse over on the loudspeaker!**

The current behavior my come from YouTube as reference: But on YouTube the volume slider space is used for the time information, so it could make sense in this case but from an accessibility standpoint this is also suboptimal. 

![2022-12-12 13_20_19-(199) G-P  Telemann - Gulliver Suite for 2 Violins - UCLA Freshman Vivaldi Conce](https://user-images.githubusercontent.com/101632236/207043641-7a50f22e-adcd-48e3-baad-294f179b86ca.png)

![2022-12-12 13_20_45-(199) G-P  Telemann - Gulliver Suite for 2 Violins - UCLA Freshman Vivaldi Conce](https://user-images.githubusercontent.com/101632236/207043674-1996eabe-3ab7-4b1a-9728-42301e41c5f6.png)

**Comment by miesgre** [comment]
Volume is controlled by keys (arrow up, arrow down).
Paella default is that the volume bar is only visible when using mouse (mouse over loudspeaker icon).

@cgruber-uos  @oas777  What is your preference?

1. volume bar only visible with mouse (mouse over loudspeaker icon).
This bar does not aport information as the loudspeaker icon change showing the audio level. Is only used to change the volume using the mouse.

2. volume bar always visible
As the loudspeaker icon already shows the sound level, the bar does not provide more information and only takes up space.

3. volume bar visible when focus on loudspeaker icon
When the focus is on the loudspeaker icon, the bar is displayed. It would be something similar to what Youtube does.

**Comment by miesgre** [comment]
We updated the demo with option `2 (volume always visible)`.
Demo: https://polimediaupv.github.io/paella-ethz/?id=belmar_16_9_hls

Check if this is what you want or we go back to option `1`

**Comment by oas777** [comment]
Same here: I can live with this version (which according to Clemens' comments is the most accessible), let's see whether he replies. Otherwise, we'll keep it until the final accessibility review by Oriane (ZHAW).
